image/image.go

```diff
@@ -263,3 +263,13 @@ func NewImgJSON(src []byte) (*Image, error) {
 	}
 	return ret, nil
 }
+
+// Check wheather id is a valid image ID or not
+func ValidateID(id string) error {
```

Comment:
mmm I don't know, the stringid/truncindex are tied but can be useful outside docker too I suppose